packages/preview/flower-book/0.2.8/README.md

```diff
@@ -0,0 +1,59 @@
+# The `FlowerBook` Package
+<div align="center">Version 0.2.8</div>
+
+A flower style template
+
+## Template adaptation checklist 模板调整清单
+
+- [ ] remove/replace the example test case
+- [ ] (add your actual code, docs and tests)
+- [ ] remove this section from the README
```

Comment:
I don't think this section should be shown on Typst Universe, it feels like the template is not ready yet. Can you remove it, please?